services/agent/docker/healthcheck.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec simcore-service-agent info
```

Comment:
VERY CAREFUL with healthchecks: we had a really bad experience with ``migration`` healthcheck. It burning down our CPU credit!

Healthcheck is a high frequency call and MUST consume almost no CPU. Here you have two layers: first the python CLI with typer (pay attention to the imports, they might consume quite a lot) and then ``services/agent/src/simcore_service_agent/info/_consumer.py``.

Reply:
dropped this approach in favour of using FastAPI.